texts/pride_prejudice.txt

```diff
@@ -0,0 +1,1833 @@
+;	"must" vbmod inf
+	"must" vbmod pres
+"<know>"
+	"know" vblex inf
+;	"know" vblex pres
+;	"know" vblex imp
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<I>"
+	"prpers" prn subj p1 mf sg
+"<am>"
+	"be" vbser pres p1 sg
+"<thinking>"
+;	"thinking" n sg
+;	"think" vblex pprs
+;	"think" vblex subs
+	"think" vblex ger
+"<of>"
+	"of" pr
+"<his>"
+;	"his" adj pos pred
+;	"his" det pos sp
+	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<marrying>"
+;	"marry" vblex pprs
+;	"marry" vblex subs
+	"marry" vblex ger
+"<one>"
+;	"one" num sg
+	"one" prn dem mf sg
+"<of>"
+	"of" pr
+
+      
+"<them>"
+;	"that" prn dem pl
+	"prpers" prn obj p3 mf pl
+"<.>"
+	"." sent
+”
+"<.>"
+	"." sent
+
+
+      “
+"<Is>"
+	"be" vbser pres p3 sg
+"<that>"
+;	"that" cnjsub
+	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<his>"
+;	"his" adj pos pred
+;	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+	"prpers" det pos p3 m sg
+"<design>"
+	"design" n sg
+;	"design" vblex inf
+;	"design" vblex pres
+;	"design" vblex imp
+"<in>"
+;	"in" adv
+	"in" pr
+"<settling>"
+;	"settle" vblex pprs
+;	"settle" vblex subs
+	"settle" vblex ger
+"<here>"
+	"here" adv
+"<?>"
+	"?" sent
+”
+"<.>"
+	"." sent
+
+
+      “
+"<Design>"
+	"design" n sg
+;	"design" vblex inf
+;	"design" vblex pres
+;	"design" vblex imp
+"<!>"
+	"!" sent
+"<Nonsense>"
+	"nonsense" n sg
+"<,>"
+	"," cm
+"<how>"
+;	"how" adv
+;	"how" preadv
+	"how" adv itg
+"<can>"
+;	"can" n sg
+;	"can" vbmod inf
+	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<you>"
+	"you" prn pers p2 mf sg
+;	"prpers" prn subj p2 mf sp
+;	"prpers" prn obj p2 mf sp
+"<talk>"
+;	"talk" n sg
+	"talk" vblex inf
+;	"talk" vblex pres
+;	"talk" vblex imp
+"<so>"
+	"so" adv
+;	"so" preadv
+;	"so" cnjadv
+"<!>"
+	"!" sent
+"<But>"
+	"but" cnjcoo
+;	"but" pr
+"<it>"
+	"prpers" prn subj p3 nt sg
+;	"prpers" prn obj p3 nt sg
+"<is>"
+	"be" vbser pres p3 sg
+"<very>"
+;	"very" adv
+	"very" preadv
+"<likely>"
+	"likely" adv
+;	"likely" adj sint     
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<he>"
+	"prpers" prn subj p3 m sg
+"<may>"
+;	"may" vbmod inf
+	"may" vbmod pres 
+"<fall>"
+;	"fall" n sg
+	"fall" vblex inf
+;	"fall" vblex pres
+;	"fall" vblex imp
+"<in love>"
+	"in love" adj
+"<with>"
+	"with" pr
+"<one>"
+;	"one" num sg
+	"one" prn dem mf sg
+"<of>"
+	"of" pr
+"<them>"
+;	"that" prn dem pl
+	"prpers" prn obj p3 mf pl
+"<,>"
+	"," cm
+"<and>"
+	"and" cnjcoo
+"<therefore>"
+	"therefore" adv
+;	"therefore" cnjadv
+"<you>"
+;	"you" prn pers p2 mf sg
+	"prpers" prn subj p2 mf sp
+;	"prpers" prn obj p2 mf sp     
+"<must>"
+;	"must" n sg
+;	"must" vbmod inf
+	"must" vbmod pres
+"<visit>"
+;	"visit" n sg
+	"visit" vblex inf
+;	"visit" vblex pres
+;	"visit" vblex imp
+"<him>"
+	"prpers" prn obj p3 m sg
+"<as soon as>"
+	"as soon as" adv
+;	"as soon as" cnjsub
+;	"as soon as" cnjadv
```

Comment:
I think this might be `<cnjadv>`, but see if the tagging guidelines say anything about this

Reply:
I have found nothing in the guidelines, but now that I think of it, it actually resembles a subordinating conjunction.

Take the following sentence: _I walked away **when** I saw him._

Here, _when_ is a subordinating conjunction. Now, if we change it by _as soon as_, basically it is affecting the sentence the same way:
_I walked away **as soon as** I saw him._

(Well, I'm not actually too sure of this, but at least it makes sense in my mind)

Reply:
Ok, makes sense to me